packages/contracts/contracts/LenderCommitmentForwarder.sol

```diff
@@ -363,30 +362,41 @@ contract LenderCommitmentForwarder is TellerV2MarketForwarder {
 
     function getRequiredCollateral(
```

Comment:
lets add a comment block for the function

Reply:
done